### PR TITLE
cleanup parseExcludedCIDRs

### PR DESCRIPTION
**Pull request by JornShen**
/kind cleanup

There has been one func parseExcludedCIDRs in k8s.io/utils/net
https://github.com/kubernetes/utils/blob/fddb29f9d0095a7a37582a1d86732ae83620f2e8/net/net.go#L30

```release-note
NONE
```

### CODE REVIEW

File: pkg/proxy/ipvs/proxier.go
Comment (aojea) [inline pkg/proxy/ipvs/proxier.go:332]: This changes the behaviour since once it finds a wrong cidr it stops processing the others, however, with this function it just continues.

https://github.com/kubernetes/utils/blob/fddb29f9d0095a7a37582a1d86732ae83620f2e8/net/net.go#L35 

 I think that @andrewsykim and @lbernail are the ones more familiar with the ipvs details, I really don't have context here

Comment (hanlins) [reply pkg/proxy/ipvs/proxier.go:332]: Interesting, in the proxy config [validation](https://github.com/kubernetes/kubernetes/blob/d6a9061cb43ab98054f0f455115d87e1b20f7971/pkg/proxy/apis/config/validation/validation.go#L51), actually it already checked the error for excluded CIDRs [(here](https://github.com/kubernetes/kubernetes/blob/d6a9061cb43ab98054f0f455115d87e1b20f7971/pkg/proxy/apis/config/validation/validation.go#L152)). It seems like a duplication logic here and we can assume the CIDRs here are just valid.

File: pkg/proxy/ipvs/proxier.go
Comment (hanlins) [inline pkg/proxy/ipvs/proxier.go:445]: I would suggest removing the validation here as we've already checked that the CIDRs are valid [here](https://github.com/kubernetes/kubernetes/blob/d6a9061cb43ab98054f0f455115d87e1b20f7971/pkg/proxy/apis/config/validation/validation.go#L304), during the validation for proxy configs.

File: pkg/proxy/ipvs/proxier.go
Comment (hanlins) [inline pkg/proxy/ipvs/proxier.go:459]: I suggest directly consume `excludeCIDRs` as it should have been validated already.

**Review by hanlins — CHANGES_REQUESTED** [review]
Comments left, also please check why the tests are failing. It seems they failed 2 out 2 times, not sure if it's temporary or consistent.

File: pkg/proxy/ipvs/proxier.go
Comment (aojea) [reply pkg/proxy/ipvs/proxier.go:332]: ahh good catch

File: pkg/proxy/ipvs/proxier.go
Comment (JornShen) [reply pkg/proxy/ipvs/proxier.go]: @hanlins  yeah, you are right. But we still need it to parse string data(excludeCIDRs) to IPNet list.

Comment (JornShen) [reply pkg/proxy/ipvs/proxier.go]: I add comment 
https://github.com/kubernetes/kubernetes/pull/99448/files#diff-1f2043db3a45960023376d03e4dd43762d1382ad7ea94e3f26c956410b749c97R445

Comment (hanlins) [reply pkg/proxy/ipvs/proxier.go]: I see, thanks.

**Comment by hanlins** [comment]
/retest
/lgtm
Seems the tests are flaky, you still need approvals from owners.

**Comment by aojea** [comment]
/assign @andrewsykim 
@uablrek maybe is interested in step up as approver, he is actively reviewing and using IPVS
So Andrew, maybe as subproject approver, you can both talk and, if he wants, nominate as IPVS approver
/cc @thockin as sig-net chair, we talked about this last sig-network meeting

**Comment by uablrek** [comment]
I can gladly take more responsibility for the ipvs proxier. I have worked with since I started with K8s. My first issues and PRs was on the ipvs proxier and I also got experience when working with the meta proxier.

**Comment by JornShen** [comment]
ping @uablrek

**Pull request merged**
